deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml

```diff
@@ -37,4 +37,5 @@ spec:
           args:
             - "--v=5"
             - "--leader-election=true"
+            # end snapshot controller args
```

Comment:
Can you add the following comments here so we remember not to remove this?
```
Adds a marker to the snapshot-controller manifests. This is needed to enable feature gates in CSI prow jobs. In https://github.com/kubernetes-csi/csi-release-tools/pull/209, the snapshot-controller YAML is updated to add --prevent-volume-mode-conversion=true so that the feature can be enabled for certain e2e tests.
```

Reply:
done